sdk/keyvault/azure-security-keyvault-secrets/README.md

```diff
@@ -197,27 +197,17 @@ System.out.printf("Secret's updated expiry time %s \n", updatedSecretProperties.
 
 ### Delete a Secret
```

Comment:
[here](https://github.com/Azure/azure-sdk-for-java/pull/6061/files#diff-65ce8a0dfab39275230ae65bdb7a6e87R188) should be `updateSecretProperties`

Reply:
updated